.github/workflows/main.yml

```diff
@@ -5,6 +5,11 @@ on:
   push:
     branches:
       - '**'
```

Comment:
I'm pretty sure that prettier runs on the markdown docs. I don't know if we made the conscious decision to disable prettier on all our documentation. By disabling it on the CI that is a decision we are making so we should discuss it explicitly with the team.

If it is decided that is the right direction then the prettier config needs to be updated to ignore these files. Otherwise errors could be introduced that are checked in and not validated by the CI build.

Reply:
After talking with the team we've decided it's not slowing us down to run the CI for doc-only changes, and we'd rather keep running prettier on our docs. Abandoning this change.